src/VirtualClient/VirtualClient.Dependencies/CUDAAndNvidiaGPUDriverInstallation.cs

```diff
@@ -400,29 +359,5 @@ private async Task CudaAndNvidiaGPUDriverInstallationOnWindowsAsync(EventContext
             await this.ExecuteCommandAsync(installerPath, "-y -s", Environment.CurrentDirectory, telemetryContext, cancellationToken)
                 .ConfigureAwait(false);
         }
-
```

Comment:
again, if remove the executeCommandAysnc here, need to add runElevated:true on line 208 and 359